platform/platform-app/pom.xml

```diff
@@ -462,7 +467,7 @@
         <cxf.resources.base.path/>
         <!-- End apache-cxf-3.1.4-src/parent/pom.xml-->
```

Comment:
This version number is meant to be 3.1.4 because that is the version that these properties were originally copied from. Matt and I updated the comment here to better explain what going on.